### PR TITLE
[Copy] Skill portfolio and Skill showcase

### LINKED ISSUE #12361: 🔤 Copy updates skills portfolio and showcase
## 🔤 Copy

We recently renamed users' skills library to skills portfolio. However, we didn't update all the other copy yet to reflect this change.

We also have been using two different translations for Skill showcase in French. I'd like us to only use "Vitrine de compétences".

### Skills portfolio

- In the **dialog** that opens when you want to add a new skill to your portfolio, please update the copy to: 

> **Find and add a skill to your portfolio**
> Search through our database of skills and link one to your profile.

> **Trouvez une compétence et ajoutez-la à votre portfolio**
> Cherchez dans notre base de données de compétences et associez-en une à votre profil.

-	Please update the button to "Sauvegardez et ajoutez cette compétence"
<details>
<summary>screenshot</summary>

image
</details>

- Under "**Skill showcase**", please update the copy to: “Your skill showcase allows you to curate lists of skills from your portfolio that present a more focused and personalized story about your strength and areas of interest. While your skills portfolio serves as a central place to manage all the skills you add to your profile, the showcase is paired with your job applications. It's shared with recruiters and managers to give them a fuller picture of your talents.
- French: Please update the heading to « Vitrine de compétences” 
- And the text underneath to : “Votre vitrine de compétences vous permet de créer des listes de compétences à partir de votre portfolio qui présentent de manière plus ciblée et personnalisée vos forces et vos domaines d’intérêt. Bien que votre portfolio serve d’endroit central pour gérer toutes les compétences que vous ajoutez à votre profil, la vitrine est jumelée à vos demandes d’emploi. Elle est partagée avec les membres du personnel de recrutement et les gestionnaires pour leur donner un tableau plus complet de vos talents."

<details>
<summary>screenshot</summary>

image
</details>

### Skill showcase

- Please update the French page title and bread crumb to "Vitrine de compétences"
<details>
<summary>screenshot</summary>

image
</details>

- Under « Compétences que vous souhaitez améliorer », please update the copy to : « Si vos principales compétences mettent… »
<details>
<summary>screenshot</summary>

image
</details>

#### Your top 5 behavioural skills
- Under « Your top 5 behavioural skills », please update the copy to: “Show managers the 5 strongest behavioural skills that make you an awesome member of the team. You can edit them at any time and are free to order the skills however you like. Skills that you add here that aren't already in your portfolio will be added for you automatically.”
- French: “Montrez aux gestionnaires les cinq compétences comportementales les plus fortes qui font de vous un membre formidable de l’équipe. Vous pouvez les modifier à tout moment et vous êtes libre de mettre les compétences dans l’ordre qui vous convient. Les compétences que vous ajoutez ici qui ne sont pas déjà dans votre portfolio seront ajoutées automatiquement. »

-	Please also change the button to “Add a skill (x/5)” and “Ajoutez une compétence (x/5)”

#### Your top 10 technical skills
-	Under « Your top 10 technical skills », please update the copy to: “Show managers the 10 technical skills that best represent your expertise and skill set. You can edit them at any time and are free to order the skills however you like. Skills that you add here that aren't already in your portfolio will be added for you automatically.”
- French: “Montrez aux gestionnaires les 10 compétences techniques qui représentent le mieux votre expertise et vos compétences. Vous pouvez les modifier à tout moment et vous êtes libre de mettre les compétences dans l’ordre qui vous convient. Les compétences que vous ajoutez ici qui ne sont pas déjà dans votre portfolio seront ajoutées automatiquement. »

- Please also change the button to “Add a skill (x/10)” and “Ajoutez une compétence (x/10)”

#### 3 behavioural skills you’d like to improve
-	Under “3 behavioural skills you'd like to improve”, please update the copy to: “Help managers understand which behavioural skills you’d like to improve. You can edit them at any time and are free to order the skills however you like. Skills that you add here that aren't already in your portfolio will be added for you automatically.” 
- French: « Aidez les gestionnaires à comprendre quelles sont les compétences comportementales que vous aimeriez améliorer. Vous pouvez les modifier à tout moment et vous êtes libre de mettre les compétences dans l’ordre qui vous convient. Les compétences que vous ajoutez ici qui ne sont pas déjà dans votre portfolio seront ajoutées automatiquement. »
- Please also change the button to “Add a skill (x/3)” and “Ajoutez une compétence (x/3)”
	
#### 5 technical skills you'd like to train
-	Under “5 technical skills you'd like to train”, please update the copy to: “Help managers understand which technical skills you'd be interested in training for. You can edit them at any time and are free to order the skills however you like. Skills that you add here that aren't already in your portfolio will be added for you automatically.”
- French: “Aidez les gestionnaires à comprendre pour quelles compétences techniques vous souhaiteriez suivre une formation. Vous pouvez les modifier à tout moment et vous êtes libre de mettre les compétences dans l’ordre qui vous convient. Les compétences que vous ajoutez ici qui ne sont pas déjà dans votre portfolio seront ajoutées automatiquement. »
-	Please also change the button to “Add a skill (x/5)” and “Ajoutez une compétence (x/5)”

### Vitrine de compétences
- Please check whether we use "Démonstration de compétences" elsewhere on the platform and change to "Vitrine de compétences". If unsure, ask @NienkeBr 
- Please update the profile download to also say "Vitrine de compétences" instead of "Démonstration de compétences".

## ✅ Acceptance Criteria

- [ ] Copy edits as described in the details have been implemented.
- [ ] We don't refer to the skills showcase as "Démonstration de compétences" but only use "Vitrine de compétences"

### PR DESCRIPTION
**Pull request by mnigh**
🤖 Resolves #12361.

## 👋 Introduction

This PR updates copy related to skill portfolio and skill showcase.

## 🧪 Testing

1. Verify copy edits as described in the linked issue details have been implemented
2. Verify **Skill showcase** in French is _Vitrine de compétences_ and not _Démonstration de compétences_

## 📸 Screenshots

<details><summary>Screenshots (20)</summary>
Screen Shot 2024-12-30 at 10 22 35
Screen Shot 2024-12-30 at 10 22 56
Screen Shot 2024-12-30 at 10 24 57
Screen Shot 2024-12-30 at 10 25 18
Screen Shot 2024-12-30 at 10 26 19
Screen Shot 2024-12-30 at 10 26 36
Screen Shot 2024-12-30 at 10 27 59
Screen Shot 2024-12-30 at 10 28 12
Screen Shot 2024-12-30 at 10 28 55
Screen Shot 2024-12-30 at 10 29 06
Screen Shot 2024-12-30 at 10 29 46
Screen Shot 2024-12-30 at 10 30 02
Screen Shot 2024-12-30 at 10 31 20
Screen Shot 2024-12-30 at 10 31 32
Screen Shot 2024-12-30 at 10 32 01
Screen Shot 2024-12-30 at 10 32 13
Screen Shot 2024-12-30 at 10 33 19
Screen Shot 2024-12-30 at 10 33 32
Screen Shot 2024-12-30 at 10 37 29
Screen Shot 2024-12-30 at 10 38 16
</details>

### CODE REVIEW

**Review by esizer — APPROVED** [review]
LGTM! :+1:

File: apps/web/src/pages/Skills/SkillPortfolioPage.tsx [hunk 1]
```diff
@@ -139,15 +139,15 @@ const SkillPortfolio = ({ userSkills, skills }: SkillPortfolioProps) => {
               <p data-h2-margin="base(x1 0)">
                 {intl.formatMessage({
                   defaultMessage:
-                    "Your skill showcase allows you to curate lists of skills from your library that present a more targeted story about your strengths and areas of interest. While your skill library acts as a central place to manage all of the skills you add to your profile, the showcases you complete are paired with your job applications to help recruiters and managers see a more complete picture of your talent.",
-                  id: "ccR/uJ",
+                    "Your skill showcase allows you to curate lists of skills from your portfolio that present a more focused and personalized story about your strength and areas of interest. While your skills portfolio serves as a central place to manage all the skills you add to your profile, the showcase is paired with your job applications. It's shared with recruiters and managers to give them a fuller picture of your talents.",
```
Comment (NienkeBr) [inline apps/web/src/pages/Skills/SkillPortfolioPage.tsx:142]: There was a typo in my issue, it should stay "strengths" with an s.

**Review by NienkeBr — COMMENTED** [review]
-	Could you change “library” to portfolio” in this dialog?
 image
-	Could you change “bibliothèque” to “portfolio” in this dialog?
 image
-	Since we’re here, could we update the button to impératif? “Ajoutez cette compétence”
 image

File: apps/web/src/pages/Skills/SkillPortfolioPage.tsx (hunk 1 above)
Comment (mnigh) [reply apps/web/src/pages/Skills/SkillPortfolioPage.tsx]: 35a937dfb1a5cfa1897f998a1fb095dc9dc33c84

## Screenshot
Screen Shot 2024-12-31 at 08 55 53

**Comment by mnigh** [comment]
> * Could you change “library” to portfolio” in this dialog?

922e3e9d4bc3f1d61ea57bfe0cdd7ace765c5ddc

## Screenshot
Screen Shot 2024-12-31 at 08 51 29

---

> * Could you change “bibliothèque” to “portfolio” in this dialog?

922e3e9d4bc3f1d61ea57bfe0cdd7ace765c5ddc

## Screenshot
Screen Shot 2024-12-31 at 08 51 48

---

> * Since we’re here, could we update the button to impératif? “Ajoutez cette compétence”

d5cd8662f93c64356e358c837deb8bedda27d7d7

## Screenshot
Screen Shot 2024-12-31 at 08 54 21

**Review by NienkeBr — APPROVED** [review]

**Pull request merged**